app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadFragment.kt

```diff
@@ -108,6 +109,23 @@ class ThreadFragment : Fragment() {
     private val permissionUtils by lazy { PermissionUtils(this) }
     private val isNotInSpam by lazy { mainViewModel.currentFolder.value?.role != FolderRole.SPAM }
 
+    private val twoPaneFragment get() = parentFragment as TwoPaneFragment
+
+    // TODO: This is probably too global as a trigger. Find something more refined?
+    private val globalLayoutListener by lazy {
+        OnGlobalLayoutListener {
+            runCatching {
+                binding.toolbar.navigationIcon?.apply {
+                    if (twoPaneFragment.areBothShown()) {
+                        if (alpha != MIN_ALPHA) alpha = MIN_ALPHA
+                    } else {
+                        if (alpha != MAX_ALPHA) alpha = MAX_ALPHA
+                    }
```

Comment:
We should remove the button entirely and not make its icon invisible

Reply:
Updated here : https://github.com/Infomaniak/android-kMail/pull/1592